src/components/Form/Form.tsx

```diff
@@ -48,13 +50,23 @@ const Form = (props: FormProps) => {
 
 	useEffect(() => {
 		setTopComponentHeight(topComponentRef.current?.offsetHeight);
-		setTopComponentDrawerHeight && setTopComponentDrawerHeight(`${topComponentRef.current?.offsetHeight}px`);
 
-		if (formContentRef?.current?.children[0] && setFormLayoutPadding) {
-			const layoutPaddingTop = window?.getComputedStyle(formContentRef.current.children[0]).getPropertyValue("padding-top");
-			const layoutPaddingBottom = window?.getComputedStyle(formContentRef.current.children[0]).getPropertyValue("padding-bottom");
-			setFormLayoutPadding(`${layoutPaddingTop} - ${layoutPaddingBottom}`);
-		}
+		dispatchRef && topComponentRef.current && dispatchRef({
+			type: "update",
+			ref: {
+				id: "topComponentDrawerRef",
+				current: topComponentRef.current
+			}
+		});
+
+		formContentRef?.current?.children[0] && dispatchRef &&

```

Comment:
This wouldn't always work as expected because formContentRef it's not in the dependency array, it should be but **not** in this useEffect, it should be in its own because then it will trigger changes in the other dispatchRef